src/text.ts

```diff
@@ -45,6 +45,15 @@ export function setTextContent(node: Node, text: string): void {
   } else if (isTextNode(node)) {
     node.value = text;
   } else if (isParentNode(node)) {
+    const children = queryAll(
```

Comment:
here i think we should actually remove all children and append the text node afterwards.

that'd match how DOM `textContent` works, too.

Reply:
That is correct, I had just copied the query from `getTextContent` before.

Now that I look back at the query, I notice that comment nodes are included. While I do think that we should be able to modify the text content from comment nodes, if the `node` is a parent node, I think they should be ignored. Again this matches the behavior of native DOM.

Reply:
i think you're right and they shouldn't be included. good catch!